app/code/Dss/LazyImageLoader/view/frontend/templates/js.phtml

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Digit Software Solutions.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the EULA
+ * that is bundled with this package in the file LICENSE.txt.
+ *
+ * @category  Dss
+ * @package   Dss_LazyImageLoader
+ * @author    Extension Team
+ * @copyright Copyright (c) 2024 Digit Software Solutions. ( https://digitsoftsol.com )
+ */
+?>
+<?php
+/**
+ * LazyImageLoader block
+ *
+ * @var  $block  \Dss\LazyImageLoader\Block\LazyLoad
+ */
+?>
+<?php
+$helper = $block->getHelper();
+$escaper->escapeHtml($helper->getFullActionName());
+$threshold = $helper->getThreshold();
+?>
+<?php if ($helper->isEnabled()): ?>
+    <style>
+        img.lazy {
+            background: url(<?= $escaper->escapeHtml($helper->getLazyImage()); ?>) no-repeat;
+            background-size: <?= $escaper->escapeHtml($helper->getLoadingWidth()); ?>px;
```

Comment:
add css from js. and remove css from phtml

Reply:
Please check i remove style from phtml file and add it by using js.